pkg/apis/monitoring/v1/types.go

```diff
@@ -87,19 +87,19 @@ type ProxyConfig struct {
 	// that should be excluded from proxying. IP and domain names can
 	// contain port numbers.
 	//
-	// It requires Prometheus >= v2.43.0.
+	// It requires Prometheus >= v2.43.0 or Alertmanager >= 0.25.0.
```

Comment:
`ProxyConfig` is reused by Prometheus and Alertmanager, so this description is changed.
/cc @simonpasquier PTAL